assets/chat/css/style.scss

```diff
@@ -1491,6 +1502,23 @@ hr {
       &.creator {
         color: $text-color1;
       }
+      &.favorite {
+        cursor: pointer;
```

Comment:
If you make the element a button, which is more semantically correct anyway, you won't have to set the cursor type. You might have to remove the default button styles if there isn't a reset somewhere in this file already, though.

Reply:
The button element has a default cursor so you'd still need to set it to cursor. I can still change it to a button element if you want.

Reply:
You know what, I'm so used to CSS frameworks setting the default cursor for buttons to pointer I assumed that was the browser default. That's pretty funny.

Sure, you can go ahead and make it a button. You can also define reset styles for buttons at the top of the file that gets rid of its background, border, and sets the cursor type, too. I believe the only other place buttons are used are in the overlay that pops up if you try to send a message while logged out. Digging through our HTML, it's alarming how many times we use anything but a button when a button is most appropriate. Something to address in the future.